shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.m

```diff
@@ -12,6 +12,19 @@
 #error ARC must be enabled!
 #endif
 
+extern NSNotificationName const FlutterViewControllerWillDealloc;
+
+/// OCMock can't be used for FlutterEngine sometimes because it retains arguments to invocations
```

Comment:
I didn't understand this comment :D

Also, this is more a comment than a class doc right? Maybe //?

Reply:
Updated the text to be more clear.  I think /// is appropriate since its intention is to be a docstring.